modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/fds.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	IFrontendDataSetProps,
+
+	// @ts-ignore
+
+} from '@liferay/frontend-data-set-web';
+
+export interface fdsItem<T> {
+	action: {id: string};
+	itemData: T;
+	openSidePanel: ({url}: {url: string}) => void;
+	value: LocalizedValue<string>;
+}
+
+export interface IField extends IFrontendDataSetProps {
```

Comment:
@alinedoleron I think we could rename this interface to be more semantic. Something like "ITableProps" or "IFDSTableProps". What do you think?